content/en/docs/v3.6/op-guide/kubernetes.md

```diff
@@ -0,0 +1,438 @@
+---
+title: Run etcd clusters as a Kubernetes StatefulSet
```

Comment:
Suggest adding a sentence and link to this new page under https://etcd.io/docs/v3.6/install.  There is a Kubernetes section on that page which is currently blank.